README.md

```diff
@@ -98,6 +98,7 @@ Options:
   [--server], [--no-server]  # Start standalone web server on port 8080
   [--port=PORT]              # Set port for --server option
   [--host=HOST]              # Set host for --server option
+  [--watch], [--no-watch]    # Reload API documentation when API Description Document changed
```

Comment:
it is generated from https://github.com/apiaryio/apiary-client/pull/142/files#diff-969dbf95872b9f711800ef6cb6b40992R28 `no-` option is added automatically for bool arg. default is false (== --no-watch)